src/main/g8/README.md

```diff
@@ -0,0 +1,41 @@
+**This is a template README.md.  Be sure to update this with project specific content that describes your performance test project.**
```

Comment:
This is something I missed in the original performance-testing-template. But it probably makes sense to add it here:

- Logging. The template uses `Logback.xml` to configure the log levels. The file is under `src/test/resources` The default log level is WARN. This can be updated to a lower level like TRACE to view the requests and response for any investigation. 

Something of this nature, but you can reword it and introduce a subheading.